azext_iot/operations/hub.py

```diff
@@ -62,7 +62,7 @@ def iot_query(
 
     try:
         query_args = [query_command]
-        query_method = service_sdk.registry_manager.query_iot_hub
+        query_method = service_sdk.query.get_twins
```

Comment:
`get_twins` is what the query API is called?

Reply:
yes. i compared the description and param, and verified the output, that's what has been renamed to